.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 lib/*
 dist
+.idea
```

Comment:
Consider handling this locally via a global gitignore: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/